packages/amplify_auth_cognito/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -38,6 +38,17 @@
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
         </activity>
+          <activity
+      android:name="com.amazonaws.mobileconnectors.cognitoauth.activities.CustomTabsRedirectActivity">
+      <intent-filter>
+          <action android:name="android.intent.action.VIEW" />
+
+          <category android:name="android.intent.category.DEFAULT" />
+          <category android:name="android.intent.category.BROWSABLE" />
+
+          <data android:scheme="myapp" />
+      </intent-filter>
```

Comment:
Will this need to be added by flutter developers in their app?

Reply:
Yes, for now - we do not know the value for `android:scheme`.  To my knowledge the only way to inject a variable into the manifest is by modifying the gradle file.

Reply:
We discussed this when we initially talked through hosted ui.